hyperspy/io_plugins/tiff.py

```diff
@@ -899,7 +923,7 @@ def _parse_string(value):
 
 mapping_fei = {
     'fei_metadata.Beam.HV':
-        ("Acquisition_instrument.SEM.beam_energy", lambda x: float(x) * 1e-3),
+        ("Acquisition_instrument.SEM.beam_energy", _parse_beam_energy_FEI),
```

Comment:
Why did you replace the lambda function with a standard function? The lambda function has the advantage that one does not need to scroll to the function to check what it does.

Reply:
This was causing another error if that key doesn't exist (which it won't, for the navigation image). In that case, `x` will be `None` or `''`, and it causes a `TypeError`. It's safer (and more future-proof) to wrap each of those conversions in a `try`/`except`.